src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -16,6 +16,10 @@
 logger = logging.getLogger(__name__)
 
 
+PV_FIELDS = 'pv_name,pv_tags,pv_uuid,vg_name,lv_uuid'
+VG_FIELDS = 'vg_name,pv_count,lv_count,snap_count,vg_attr,vg_size,vg_free,vg_free_count'
```

Comment:
This is again redefined further down in this file. https://github.com/ceph/ceph/blob/6968131d8ef61991dd907f4376875bd4fc7ef3ae/src/ceph-volume/ceph_volume/api/lvm.py#L533

One of the issues with style cleanups like this.